google-cloud-datastore/src/test/java/com/google/cloud/datastore/it/ITDatastoreTest.java

```diff
@@ -341,12 +373,14 @@ public void testRunGqlQueryNoCasting() throws InterruptedException {
     Query<Entity> query1 = Query.newGqlQueryBuilder(ResultType.ENTITY, "select * from " + KIND1)
         .setNamespace(NAMESPACE)
         .build();
-    QueryResults<Entity> results1 = DATASTORE.run(query1);
-    while (Iterators.size(results1) < 1) {
-      Thread.sleep(500);
-      results1 = DATASTORE.run(query1);
-    }
-    results1 = DATASTORE.run(query1);
+    Query<Entity> scQuery1 = Query.newEntityQueryBuilder()
```

Comment:
The newEntityQueryBuilder() itself doesn't not guarantee strongly consistent query. It is the "ancestor filter" that makes the query an ancestor query, which is guaranteed to be strongly consistent.  A comment is added to make this clear.